docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,116 @@
+### How we communicate product ideas
+We are a remote company, so it’s very important that we have common knowledge and understanding. Introducing a communication strategy that is more agile and allows for open communication is one way we will tackle this. 
+That means that product initiatives will collaborate with research, engineering, and design right from the beginning to make sure we have shared context where it counts most.
+We’ll also focus on product discovery, rapid prototyping, and validation so that we can prioritize effectively. We’ll make sure to do MVPs, described as Minimum Viable _Prototypes_ in Marty Cagan’s book _Inspired_.  This is really a difference in functionality and fidelity. A product will typically have many features at high fidelity, while a prototype will have limited functionality at a lower fidelity. 
+We typically think of MVPs and Minimum Viable Products, but that often means that we are creating something that’s very well thought out and crafted _before_ we check to make sure it’s something people want or something we should spend our time doing. By focusing on research, discovery, and _prototyping_ we keep the focus on our end users and retain the ability to rapidly test potential solutions.
+### How we focus on personas
+Because we want to stay user-focused, we will also be doing discovery and usability research against specific personas that are important to Gatsby. As we learn more about our audience, we’ll get a better understanding of current personas and start to see new personas that we should define.
+How we prioritize
+Since we’ll be doing a lot of discovery, usability research, and prototyping, that means we’ll have quite a bit of prioritization to do.
+
+We’ll prioritize based on OKRs we talked about before. Objectives show what we want to accomplish and key results show how we’ll measure that we’ve accomplished it.
+
+The business vision and objective is set at the company level, and the business results are set by the individual teams. This means that teams will have the autonomy to hit the business objective in the best way they see fit.
+How we put it all together 
+Outcome-based roadmaps
+
+Once we understand the ideas we want to prioritize, the objectives we’re trying to hit, and the results we’ll use to measure those outcomes, we can put together a roadmap that shows where we want to focus.
+
+Typical roadmaps are difficult to use because they don’t communicate information in a way that addresses everyone’s needs.
+
+To make sure that we don’t fall into this trap, we will do outcome-based roadmaps. Our roadmap will include:
+our business vision
+our objectives
+our business results
+
+This will help us focus our roadmaps on business problems we’re trying to solve, rather than a general feature that we’re not sure is important.
+
+
+Where will all of this information live?
```

Comment:
looks like this needs some formatting help

Reply:
Made the formatting updates to this section.